Sources/StreamChatUI/Composer/ComposerVC.swift

```diff
@@ -731,10 +760,7 @@ open class _ComposerVC<ExtraData: ExtraDataTypes>: _ViewController,
         _ picker: UIImagePickerController,
         didFinishPickingMediaWithInfo info: [UIImagePickerController.InfoKey: Any]
     ) {
-        var completion: (() -> Void)?
-        defer {
-            picker.dismiss(animated: true, completion: completion)
-        }
+        picker.dismiss(animated: true, completion: nil)
```

Comment:
does showing max size alert work fine without waiting for a picket to be dismissed?
```suggestion
        picker.dismiss(animated: true) {
            //do the processing after the picket is dismissed?
        }
```

Reply:
Yes, better to wait for its complete dismissal. Changed it